src/i18n/en.js

```diff
@@ -10,7 +10,8 @@ export default {
     login: {
       blocked_user: 'The user is blocked.',
       invalid_user_password: 'Wrong credentials.',
-      invalid_captcha: 'The text you entered was incorrect. <br /> Please try again.',
+      invalid_captcha: 'Solve the challenge question to verify you are not a robot.',
+      invalid_recaptcha: 'Select the checkbox to verify you are not a robot.',
```

Comment:
We may need to provide additional translations for these, at least for the supported EU countries. I will ping you an internal conversation for context.

Reply:
done!